pages/news/2020-10-26-new-in-2-3-deck-clone.html

```diff
@@ -0,0 +1,58 @@
+title: New In 2.3: Deck Cloning
+author: Matthew Nicholson
+date: 2020-10-26 08:12:00
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown %}
+
+
+Deck cloning is a highly anticipated feature added in Mixxx 2.3. In short, it does what it says on the tin. When activated, the currently loaded song, playback state, pitch adjustment, and any playing loop will be cloned from one deck to another. This opens up a number of creative mixing possibilities, some of which we will explore below. Note: in some other DJ software, this feature is known as "instant doubles".
```

Comment:
```suggestion
Deck cloning (also called "instant doubles" in some other DJ software) is a highly anticipated feature added in Mixxx 2.3. In short, it does what it says on the tin. When activated, the currently loaded song, playback state, pitch adjustment, and any playing loop will be cloned from one deck to another. This opens up a number of creative mixing possibilities, some of which we will explore below.
```

Starting a sentence with "Note:" in a paragraph is odd.